src/App.tsx

```diff
@@ -1,17 +1,139 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import './App.scss';
 
 import users from './api/users';
+import todos from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+const preparedTodos: Todo[] = todos.map((todo) => ({
+  ...todo,
+  user: users.find((customer) => (
+    customer.id === todo.userId)) || null,
+}));
 
 const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [name, setName] = useState('');
+  const [todosToRender, setTodosToRenter] = useState(preparedTodos);
+  const [errorName, setErrorName] = useState(false);
+  const [errorTitle, setErrorTitle] = useState(false);
+
+  const checkName = (nameField: string) => {
+    if (nameField === '') {
+      setErrorName(true);
+    } else {
+      setErrorName(false);
+    }
+  };
+
+  const checkTitle = (titleField: string) => {
+    if (titleField.trim() === '') {
+      setErrorTitle(true);
+    } else {
+      setErrorTitle(false);
+    }
+
+    const exceptionSymbols = /[^A-Za-z0-9А-ЯЄІЇа-яєії\s]/gi;
+    const correctededTitle = titleField.replace(exceptionSymbols, '');
+
+    setTitle(correctededTitle);
```

Comment:
Why do you call setTitle on checkTitle method?
you don’t need set title on submit